examples/passthrough/passthrough.bal

```diff
@@ -43,12 +47,14 @@ service hello on new http:Listener(9092) {
         methods: ["POST", "PUT", "GET"],
         path: "/"
     }
-    resource function helloResource(http:Caller caller, http:Request req) {
+    resource function helloResource(http:Caller caller, http:Request req) returns error? {
         http:Response res = new;
         res.setPayload("Hello World!");
         var result = caller->respond(res);
         if (result is error) {
            log:printError("Error sending response", err = result);
+            return result;
```

Comment:
I am not sure returning the error make sense here. Because at the end of the day, we are trying to resend the message ?

Reply:
Agree. Will fix that.

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/12644/commits/13f67f9ec008a6b2f471aeb759c17b1483c3f75c